### PR TITLE
Compare snap shot ignore screen scale 

### PR DESCRIPTION
**Pull request by thanasak-s**
According to this issue [#223 ](https://github.com/ashfurrow/Nimble-Snapshots/issues/223)

### CODE REVIEW

**Review by ashfurrow — APPROVED** [review]
Yeah this makes sense, thanks for the PR! I'll get this merged today and released hopefully today/tomorrow 👍

File: Nimble_Snapshots/DynamicSize/DynamicSizeSnapshot.swift [hunk 1]
```diff
@@ -152,7 +152,8 @@ public func haveValidDynamicSizeSnapshot(named name: String? = nil,
                                          usesDrawRect: Bool = false,
                                          pixelTolerance: CGFloat? = nil,
                                          tolerance: CGFloat? = nil,
-                                         resizeMode: ResizeMode = .frame) -> Predicate<Snapshotable> {
+                                         resizeMode: ResizeMode = .frame,
+                                         isIgnoreScale: Bool = false) -> Predicate<Snapshotable> {
```
Comment (hfehrmann) [inline Nimble_Snapshots/DynamicSize/DynamicSizeSnapshot.swift:156]: At least for me, the parameter name does not make sense. A better name could be `shouldIgnoreScale`, `isIgnoringScale` or `ignoreScale` (the first one being the one I like more).

**Review by hfehrmann — COMMENTED** [review]
The comment apply for all the PR. What do you think?

File: Nimble_Snapshots/DynamicSize/DynamicSizeSnapshot.swift (hunk 1 above)
Comment (ashfurrow) [reply Nimble_Snapshots/DynamicSize/DynamicSizeSnapshot.swift:156]: Yeah that's a fair point. `shouldIgnoreScale` makes sense to me.

**Comment by ashfurrow** [comment]
@thanasak-s if you'd like to amend this pull request to use the `shouldIgnoreScale` then I can run the tests locally and get this merged/deployed 👍

**Review by ashfurrow — APPROVED** [review]
Looks good, thanks for the PR and thanks for your patience! I'll get this merged + released today.

**Comment by ashfurrow** [comment]
Alright, thanks again! This has been released as version `9.1.1`.

**Pull request merged**